packages/node/src/models/proposed-app-instance-info.ts

```diff
@@ -16,7 +16,7 @@ export interface IProposedAppInstanceInfo {
   myDeposit: BigNumber;
   peerDeposit: BigNumber;
   timeout: BigNumber;
-  initialState: AppState;
+  initialState?: AppState;
```

Comment:
Wait, why is this being made possibly undefined?

Reply:
Because it may be possibly undefined when it gets passed in...as in the initial state has literally not been defined?